app/build.gradle

```diff
@@ -91,11 +91,11 @@ android {
     }
 }
 
-def daggerVersion = '2.19'
+def daggerVersion = '2.23.2'
 def okhttpVersion = '3.11.0'
-def retrofitVersion = '2.4.0'
+def retrofitVersion = '2.6.1'
 def assertjVersion = '1.2.0'
-def acraVersion = '5.1.3'
+def acraVersion = '5.4.0'
```

Comment:
updating library version is very good. Thanks. Could have been done in a separate commit thought.